src/store.rs

```diff
@@ -201,6 +211,13 @@ named!(parse_position<&[u8], OutputArg>, do_parse!(
 	>> (OutputArg::Position(x, y))
 ));
 
+named!(parse_rotation<&[u8], OutputArg>, do_parse!(
+	tag!("rotation")
```

Comment:
Could we use the same syntax as sway for this?

Reply:
Yeah I initially did but decided to split it as I was only doing rotation in a first run and sway combines rotation and other transforms into one string whereas monitors.xml doesn't. 
If the design philosophy you want is more sway-centric and anything else is just a bonus, I'm more than happy to switch that back. Will probably be able to merge in flipping while I'm at it but I doubt monitors.xml support will be all that simple.